examples/rust/dna/Cargo.toml

```diff
@@ -7,7 +7,7 @@ license = "MIT OR Apache-2.0"
 publish = false
 
 [dependencies]
-rerun = { path = "../../../crates/rerun", features = ["web_viewer"] }
+rerun = { path = "../../../crates/rerun", features = ["native_viewer"] }
```

Comment:
why did this one lose the web?

Reply:
`dna` _always_ uses `native_viewer::spawn`, never web